docs/content/management-bundles.md

```diff
@@ -852,6 +878,7 @@ bundles:
 Multiple AWS credential providers can be configured. OPA will follow an *internally defined* order to try each of the credential provider given in the configuration till success. Following order of precedence is followed when multiple credential provider is given in the configuration
 
 1. Environment Credential
+2. Assume Role Credential
```

Comment:
I'm certain the markdown renders to a perfectly numbered list here (no double 2s), but for the sake of my OCD, could we update the numbering on these entries? 😄
I believe it's possible to just have `1.` on all entries, if we want to do some - probably unnecessary - future proofing.

Reply:
Updated.